File: library/src/main/java/net/dobnikar/foregroundmanager/ActivityResumedListener.java
package net.dobnikar.foregroundmanager;

import android.app.Activity;
import android.app.Application;
import android.os.Bundle;

/**
 * Created by dejandobnikar on 19/04/2017.
 */

abstract class ActivityResumedListener implements Application.ActivityLifecycleCallbacks {
    @Override public void onActivityCreated(Activity activity, Bundle savedInstanceState) {
        // nothing
    }

    @Override public void onActivityStarted(Activity activity) {
        // nothing
    }

    @Override public void onActivityPaused(Activity activity) {
        // nothing
    }

    @Override public void onActivityStopped(Activity activity) {
        // nothing
    }

    @Override public void onActivitySaveInstanceState(Activity activity, Bundle outState) {
        // nothing
    }

    @Override public void onActivityDestroyed(Activity activity) {
        // nothing
    }
}

File: library/src/main/java/net/dobnikar/foregroundmanager/ActivityResumedListener.kt
package net.dobnikar.foregroundmanager

import android.app.Activity
import android.app.Application.ActivityLifecycleCallbacks
import android.os.Bundle

/**
 * Created by dejandobnikar on 19/04/2017.
 */
internal abstract class ActivityResumedListener : ActivityLifecycleCallbacks {
    override fun onActivityCreated(activity: Activity, savedInstanceState: Bundle?) {
        // nothing
    }

    override fun onActivityStarted(activity: Activity) {
        // nothing
    }

    override fun onActivityPaused(activity: Activity) {
        // nothing
    }

    override fun onActivityStopped(activity: Activity) {
        // nothing
    }

    override fun onActivitySaveInstanceState(activity: Activity, outState: Bundle?) {
        // nothing
    }

    override fun onActivityDestroyed(activity: Activity) {
        // nothing
    }
}
